events/wmt23.md

```diff
@@ -198,43 +239,45 @@ In 2022, the *News* machine translation task was renamed the *General* machine t
 | 17:15 - 17:30 | [**The Devil Is in the Errors: Leveraging Large Language Models for Fine-grained Machine Translation Evaluation**](https://aclanthology.org/2023.wmt-1.100.pdf) <br>Patrick Fernandes, Daniel Deutsch, Mara Finkelstein, Parker Riley, André Martins, Graham Neubig, Ankush Garg, Jonathan Clark, Markus Freitag, Orhan Firat |
 
 
-## Scientific papers
-
-### Topics
-
-- Machine translation models (neural, statistical etc. )
-- Analysis of neural models
-- Using comparable corpora
-- Selection and preparation of data
-- Semi-supervised and unsupervised learning for machine translation, transfer learning
-- Multilingual machine translation
-- Incorporating linguistic information into machine translation
-- Machine translation inference
-- Manual and automatic methods for evaluating machine translation
-- Quality estimation
-
-### Research Papers
-
-> Research papers should describe original research corresponding to the categories listed above. Research papers that have been or will be submitted to other meetings or publications must indicate this at submission time, and must be withdrawn from the other venues if accepted and published at WMT 2022.
-
-> We will not accept for publication papers that overlap significantly in content or results with papers that have been or will be published elsewhere. It is acceptable to submit work that has been made available as a technical report (or similar, e.g. in arXiv) without citing it.
-
-> For the research track, papers should be anonymised, be between 6 and 10 pages in length (excluding references) and may include supplementary material.
+## Results
 
+Full results of the shared tasks: 
```

Comment:
Hey @cefoo! I've noticed we keep talking about "Full results of the shared tasks" here, but the link we share is only for the news/general translation task. In the past (excluding 2022), the file we refer to used to cover not just the news/general translation task but also a bunch of other translation tasks in the findings. However, starting from 2022, each shared task's results are in separate files.
Here's my suggestion: we could either share links to all shared task results like I did here and also update all the previous year pages accordingly. Or, we could just jump straight into the news/general task and provide the link to the full results for that shared task. I'm leaning towards the latter option since we mostly concentrate on the news/general task. What do you think?

Reply:
Hi @liashahnazaryan!

Thank you so much for your PR!

Yes, I agree with you. We mainly focus on the General/News task, so let's keep sharing those results, and link the rest, as you suggested.

Reply:
Dear @cefoo, I've updated all the articles, where the results file includes only the general/news task. For WMT06-17, the file includes results for all shared tasks, so I left the articles for those years unchanged.